src/Mutator/Regex/PregMatchRemoveDollar.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Mutator\Regex;
+
+/**
+ * @internal
+ */
+final class PregMatchRemoveDollar extends AbstractPregMatch
+{
+    public const ANALYSE_REGEX = '/^([\/#~+])([^$]*)([$]?)\1([gmixXsuUAJD]*)$/';
```

Comment:
So we won't change an expression if someone decides to use a colon as a delimiter?